src/gridstack.js

```diff
@@ -379,7 +383,8 @@
         this.opts = _.defaults(opts || {}, {
             width: parseInt(this.container.attr('data-gs-width')) || 12,
             height: parseInt(this.container.attr('data-gs-height')) || 0,
-            item_class: 'grid-stack-item',
+            item_class: opts.item_class,
```

Comment:
That's not true because two lines above this change is:
opts.item_class = opts.item_class || 'grid-stack-item';